daml-lf/archive/da/daml_lf.proto

```diff
@@ -19,7 +18,7 @@ message ArchivePayload {
     daml_lf_0.Package daml_lf_0 = 1;
     daml_lf_1.Package daml_lf_1 = 2;
 
-    daml_lf_dev.Package daml_lf_dev = 9999;
```

Comment:
please reserve `9999`, and add a comment

Reply:
9cadc34e75

Reply:
bf29e0e97910ac043ca13406cabf0259f8e98aca